### ISSUE TITLE
Augury breaks whatsapp web

### ISSUE THREAD

**Issue by tschuege** [rangle/augury#763]
Whenever I enable Augury on chrome the web version of whatsapp does not work anymore. 

Console output of whatsapp:
![consolewhatsappwithaugury](https://cloud.githubusercontent.com/assets/6144934/19846727/0c779176-9f42-11e6-827b-05aa372efb15.png)

- Augury 1.2.3
- Chrome: Version 53.0.2785.143 (64-bit)
- Ubuntu 16.04 LTS

**Comment by mobinni** [comment]
@tschuege Tested on Chrome 54.0.2840.87 (64-bit) with Augury 1.2.3. I could not seem to reproduce it, could it be possible that something else is conflicting with Augury? What specific plugins does your browser have?

**Comment by tschuege** [comment]
@mobinni I just tried to reproduce the error. Last time it worked perfectly by just enabling and disabling augury. This time I could not reproduce it that way. So I disabled all extensions and reenabled one after another. And this time it was another extensions that caused the problem. I could enable it and whatsapp would not load and disable to fix the issue. Therefore I guess that the problem is not related to augury in particular, but to extensions in general.
Thanks for your help.
